docs/examples/loaders/GLTFLoader.html

```diff
@@ -137,6 +137,9 @@ <h3>[method:null setDRACOLoader]( [param:DRACOLoader dracoLoader] )</h3>
 		<div>
 		[page:DRACOLoader dracoLoader] — Instance of THREE.DRACOLoader, to be used for decoding assets compressed with the KHR_draco_mesh_compression extension.
 		</div>
+		<div>
+		Refer to this [link:https://github.com/mrdoob/three.js/tree/dev/examples/js/libs/draco#readme page] for the details of Draco and its decoder.
```

Comment:
minor nit: would say "Refer to this readme" rather than "Refer to this page" to get a more descriptive link keyword.

Reply:
Updated. Actually I was considering between page and readme, thanks.